ui/src/components/Usage.vue

```diff
@@ -252,7 +252,27 @@
 											)
 										}}
 									</td>
-									<td></td>
+									<td>
+										<button
+											v-if="id === 'free-100g'"
```

Comment:
Can we put this in a computed property?

Reply:
I moved them into two methods because these are inside of the v-for loop